### PR TITLE
Stories/verification pdf permissions 177516142

### PR DESCRIPTION
**Pull request by kwiliarty**

### CODE REVIEW

File: app/models/role.rb [hunk 1]
```diff
@@ -335,6 +334,13 @@ def self.permissions_with_descriptions
           'Client Extras',
         ],
       },
+      can_generate_homeless_verification_pdfs: {
+        description: '',
```
Comment (eanders) [inline app/models/role.rb]: This needs a description.

Comment (eanders) [reply app/models/role.rb]: Maybe: Allows access to generate and view homeless verification PDFs for any client for which the user has access.

File: app/models/role.rb [hunk 2]
```diff
@@ -864,155 +870,129 @@ def self.health_permissions_with_descriptions
...
       },
     }
   end
 
   def self.ensure_permissions_exist
     Role.permissions.each do |permission|
-      unless ActiveRecord::Base.connection.column_exists?(:roles, permission)
-        ActiveRecord::Migration.add_column :roles, permission, :boolean, default: false
-      end
+      ActiveRecord::Migration.add_column :roles, permission, :boolean, default: false unless ActiveRecord::Base.connection.column_exists?(:roles, permission)
```
Comment (eanders) [inline app/models/role.rb]: Can you adjust to:
```
ActiveRecord::Migration.add_column(:roles, permission, :boolean, default: false) unless ActiveRecord::Base.connection.column_exists?```

Lack of parens makes this really unclear what's going on

**Review by eanders — CHANGES_REQUESTED** [review]
@kwiliarty , this looks great, just a couple of small tweaks listed inline.

File: app/models/role.rb (hunk 1 above)
Comment (kwiliarty) [reply app/models/role.rb]: 🤦 ✅

File: app/models/role.rb (hunk 2 above)
Comment (kwiliarty) [reply app/models/role.rb]: ✅

**Comment by kwiliarty** [comment]
I've addressed the changes and pushed up a rebased branch that resolves a couple merge conflicts.

**Pull request merged**